File: C/Repetitivas/FOR/par_impar.c
#include <stdio.h>
int main()
{
    int x, n,i;

    printf("Quantos numeros voce vai digitar?");
    scanf("%d", &n);

        for(i=1; i<=n; i++){
            printf("Digite um numero:");
            scanf("%d", &x);
                if (x == 0){
                    printf("NULO! \n");
                }else if(x < 0 && x % 2 != 0){
                    printf("IMPAR NEGATIVO! \n");
                }else if(x < 0 && x % 2 == 0){
                    printf("PAR NEGATIVO! \n");
                }else if(x > 0 && x % 2 != 0){
                    printf("IMPAR POSITIVO! \n");
                }else if(x > 0 && x % 2 == 0){
                printf("PAR POSITIVO! \n");
                }
            }
}

File: Phyton/Repetitivas/For/par_impar.py
n = int(input("Quantos numeros voce vai digitar? "))

for i in range(0,n):
    x = int(input("Digite um numero: "))
    if(x > 0 and x % 2 == 0):
        print("PAR POSITIVO !")
    elif(x > 0 and x % 2 != 0):
        print("IMPAR POSITIVO !")
    elif(x < 0 and x % 2 != 0 ):
        print("IMPAR NEGATIVO !")
    elif(x < 0 and x % 2 == 0):
        print("PAR NEGATIVO !")
    else:
        print("NULO!")

